cdap-ui/app/cdap/main.js

```diff
@@ -171,6 +213,114 @@ class CDAP extends Component {
         </div>
       );
     }
+    const container = (
+      <div className="container-fluid">
+        <Switch>
+          <Route
+            exact
+            path="/"
+            render={(props) => (
+              <ErrorBoundary>
+                <RouteToNamespace {...props} />
+              </ErrorBoundary>
```

Comment:
rather than wrapping all components individually with ErrorBoundary, can we just create a wrapper component for Route that will add this ErrorBoundary implicitly?

Reply:
Added error boundary outside `Switch`